docs/samples/line/labelVisibility.md

```diff
@@ -23,29 +23,57 @@ const data = {
 };
 // </block:setup>
 
-// <block:annotation:1>
-const annotation = {
+// <block:annotation1:1>
+const annotation1 = {
   type: 'line',
   borderColor: 'lightGreen',
-  borderDashOffset: 0,
   borderWidth: 10,
-  drawTime: 'beforeDatasetsDraw',
   label: {
     enabled: false,
     backgroundColor: 'green',
-    borderWidth: 0,
     drawTime: 'afterDatasetsDraw',
-    color: 'white',
-    content: (ctx) => ['Average of dataset', 'is: ' + average(ctx).toFixed(3)],
-    textAlign: 'center'
+    content: (ctx) => ['Average of dataset', 'is: ' + average(ctx).toFixed(3)]
   },
   scaleID: 'y',
   value: (ctx) => average(ctx),
+  // For simple property changes, you can directly modify the annotation
+  // element's properties then call chart.draw().  This is faster.
+  enter({chart, element}, event) {
+    element.options.label.enabled = true;
+    chart.draw();
+  },
+  leave({chart, element}, event) {
+    element.options.label.enabled = false;
+    chart.draw();
+  }
+};
+// </block:annotation>
```

Comment:
I think it should be changed the annotation tag to `// </block:annotation1>` otherwise it's shown in the example source  as comment.

Reply:
Fixed. Thanks.